client/my-sites/stats/stats-purchase/index.tsx

```diff
@@ -0,0 +1,35 @@
+import config from '@automattic/calypso-config';
+import { useTranslate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
+import JetpackColophon from 'calypso/components/jetpack-colophon';
+import Main from 'calypso/components/main';
+import { useSelector } from 'calypso/state';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import PageViewTracker from '../stats-page-view-tracker';
+import StatsPurchaseWizard from './stats-purchase-wizard';
+
+const StatsPurchasePage = () => {
+	const translate = useTranslate();
+
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
+	const isPurchaseEnabled = config.isEnabled( 'stats/paid-stats' );
+
+	return (
+		<Main fullWidthLayout>
+			<DocumentHead title={ translate( 'Jetpack Stats' ) } />
+			<PageViewTracker path="/stats/participation/:site" title="Stats > Purchase" />
+			<div className="stats">
+				{ isPurchaseEnabled ? (
+					<StatsPurchaseWizard siteSlug={ siteSlug } />
+				) : (
+					<p>This page is empty.</p>
+				) }
```

Comment:
```suggestion
				<StatsPurchaseWizard siteSlug={ siteSlug } />
```

We can simplify this by not registering the route if the feature flag is disabled.

Reply:
I removed this paragraph by adding a redirect, but there is also a feature flag check on the route.